examples/js/local.js

```diff
@@ -33,13 +33,25 @@ const uploadFiles = () => {
 
 document.addEventListener('DOMContentLoaded', uploadFiles);
 
-// display uploaded image
-const loadMap = (image) => {
+// <-Ignore --- delete this comment
+// this function is a candidate from modularization. It's used in many other .js files
+const placeImage = (imageUrl, options = {}) => {
   map.whenReady(function() {
-    img = L.distortableImageOverlay(image, {
-      selected: true,
-      fullResolutionSrc: 'large.jpg',
-    }).addTo(map);
+    // constructs new map 
+    const noOptions = Object.keys(options).length === 0 && options.constructor === Object
+    if (noOptions) {
+      img = L.distortableImageOverlay(imageUrl, {
+        selected: true,
+        fullResolutionSrc: 'large.jpg',
+      }).addTo(map);
+    } else {
+      // reconstructs map into previous state
+      image = L.distortableImageOverlay(imageUrl, {
+        height: options.height,
```

Comment:
We'll have to make the same changes to local.js

Reply:
Sure... okay.